### PR TITLE
plugins admin UI enhancement

### LINKED ISSUE #5680: Plugins Admin is resizable and maximizable
The Plugins Admin should be resizable and maximizable.

Actual Behavior
The dialog size is frozen

Expected Behavior
The dialog must be resizable with mouse. The dialog must have a button to maximize the dialog.

### PR DESCRIPTION
**Pull request by cmeriaux**
The plugins admin is now resizable and maximisable.

Fixes #5680

### CODE REVIEW

**Comment by cmeriaux** [comment]
Now the plugin admin show the release date of plugins
Fixes #5742

image

**Comment by cmeriaux** [comment]
Fixes #5680
Fixes #5742 
Fixes #5760

File: PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp [hunk 1]
```diff
@@ -985,6 +985,100 @@ bool PluginViewList::restore(const generic_string& folderName)
 	return false;
 }
 
+void PluginViewList::setSortKey(int key_index)
+{
+	if (key_index == COLUMN_PLUGIN)
+		_sortKey = SORT_BY_NAME;
+	else if (key_index == COLUMN_VERSION)
```
Comment (donho) [inline PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp]: Sorting version of different plugins is pointless to me.

**Comment by donho** [comment]
Please make each pr per commit.

**Comment by chcg** [comment]
@donho Maybe a short note which parts are acceptable and which are not would be helpful or is this PR just closed due to the combination of multiple related topics in one PR?

**Comment by cmeriaux** [comment]
@donho @chcg Is it better to squash my commit or to submit several PR ? 
Several PR might diverge while you integrate it.

**Comment by donho** [comment]
plugin admin UI resizable commit has nothing to do with the other added features.
I need to try (to compile the PR) in order to evaluate your PR.
So Please keep them as separated as possible:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#pull-requests

**Pull request closed without merging**